cmd/web-handlers.go

```diff
@@ -1037,6 +1044,8 @@ func toWebAPIError(err error) APIError {
 		apiErrCode = ErrReadQuorum
 	case PolicyNesting:
 		apiErrCode = ErrPolicyNesting
+	case NotImplemented:
+		apiErrCode = ErrNotImplemented
```

Comment:
For the original issue, the error message now shown is:
![image](https://user-images.githubusercontent.com/200059/26901495-e8ffb796-4b8a-11e7-82e6-51e08b428e4d.png)

It says that functionality is not implemented, but the phrasing can be made more general. The missing functionality here is not really related to headers.